app/js/components/demographics.jsx

```diff
@@ -221,7 +222,13 @@ export default class Demographics extends React.Component {
             <div className='form-group demo'>
               <label className='control-label col-sm-3'> DOB: </label>
               <div className='col-sm-9'>
-                <h5>{this.state.birthdate}</h5>
+                <div className='col-sm-6'>
+                  <h5>{this.state.birthdate}</h5>
+                </div>
+                <div>
+                <FontAwesome className="fa fa-pencil-square-o" 
```

Comment:
Consider using a different format here ie. 

>`<FontAwesome`
`     className="fa fa-pencil-square-o"`
`     onClick={this.handleChange}`
>` /> `

Looks neater. Also, do not leave empty spaces at the end of any of the lines.

Reply:
Remember to indent the two middle lines too.